modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/typescript/index.ftl

```diff
@@ -1,46 +1,29 @@
-import localVarRequest from 'request';
-
-<#list schemaMap?keys?sort as key>
-	import {${key}} from './${key?uncap_first}';
-</#list>
-
-<#list schemaMap?keys?sort as key>
-	export * from './${key?uncap_first}';
+<#list schemas?keys?sort as schema>
+	import {${schema}} from './models/${schema}';
 </#list>
 
 /**
  * @author ${configYAML.author}
  * @generated
  */
 
-export interface RequestDetailedFile {
-	options?: {
-		contentType?: string;
-		filename?: string;
-	};
-	value: Buffer;
-}
-
-/* tslint:disable:no-unused-variable */
-const primitives = [
-	'string',
-	'boolean',
-	'double',
-	'integer',
-	'long',
-	'float',
-	'number',
-	'any',
-];
+<#list apiContexts?sort_by("className") as apiContext>
+	export {${apiContext.className}} from './apis/${apiContext.className}';
+</#list>
 
-const typeMap: {[index: string]: any} = {
-<#list schemaMap?keys?sort as key>
-	${key},
+<#list schemas?keys?sort as schema>
+	export {${schema}} from './models/${schema}';
 </#list>
-};
 
-function startsWith(str: string, match: string): boolean {
-	return str.substring(0, match.length) === match;
+export class HttpError extends Error {
```

Comment:
In a follow up PR remove this class and just generate the `throw new Error(...)` inline for simplicity.